src/main/java/teammates/ui/webapi/action/SearchInstructorsAction.java

```diff
@@ -26,13 +29,34 @@ public void checkSpecificAccessControl() {
         }
     }
 
+    private void addAdditionalSearchFields(InstructorsData instructorsData, List<InstructorAttributes> instructors) {
```

Comment:
Remember to write a test for it.

Reply:
it's a private method so i checked for it in the existing classes if thats ok

Reply:
Of course, you are writing action-level test **NOT** unit-level test.

Reply:
Also, if you are adding `institute`, just call the method as `addInstitueField`